src/types/query.d.ts

```diff
@@ -66,20 +154,48 @@ export module query {
   export function IsCredentials(expr: ExprArg): Expr
   export function IsRole(expr: ExprArg): Expr
 
-  export function Get(ref: ExprArg, ts?: ExprArg): Expr
+  export function Get<T extends object>(
+    ref: Expr.Ref<T>,
```

Comment:
`Get` can either receive a `Ref` or a `SetRef`

Reply:
correct, this is how it is already, but will need to be updated when `SetRef` is changed as you suggested earlier